### PR TITLE
Wording for CREATE DIRECTORY and MERGE DIRECTORY

### PR DESCRIPTION
**Pull request by dwicinas**

### CODE REVIEW

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/22_create_directory.mdx [hunk 1]
```diff
@@ -22,7 +22,11 @@ CREATE DIRECTORY <name> AS '<pathname>'
 
 ## Description
 
-The `CREATE DIRECTORY` command creates an alias for a file system directory pathname. You must be a database superuser to use this command.
+To create directories, you must have the `CREATE ANY DIRECTORY` system privilege.
+
+When you create a directory, you are automatically granted the `READ`, `WRITE`, and `EXECUTE` privileges on the directory, and you can grant `READ`, `WRITE`, and `EXECUTE` privileges to other users and roles. The superuser can also grant these privileges to other users and roles.
```
Comment (ajit-at-edb) [inline product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/22_create_directory.mdx:27]: There are only Read and Write privileges on directory. There is no execute privilege

**Comment by ajit-at-edb** [comment]
Label should contain Create and Drop directory. Please remove (Merge directory)

**Comment by ajit-at-edb** [comment]
Should we mention about how to grant/revoke create/drop any directory to/from user/role commands

File: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/22_create_directory.mdx (hunk 1 above)
Comment (dwicinas) [reply product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_sql/22_create_directory.mdx:27]: Removed `EXECUTE'

**Comment by dwicinas** [comment]
> Label should contain Create and Drop directory. Please remove (Merge directory)

@ajit-at-edb I don't understand what you're referring to. Can you be more explicit? Thanks.

**Comment by dwicinas** [comment]
> Should we mention about how to grant/revoke create/drop any directory to/from user/role commands

@ajit-at-edb I think that's a good idea but I would need some guidance on what instructions to provide.

**Comment by ajit-at-edb** [comment]
> > Should we mention about how to grant/revoke create/drop any directory to/from user/role commands
> 
> @ajit-at-edb I think that's a good idea but I would need some guidance on what instructions to provide.

Please find below sample commands commands


grant/revoke create any directory to/from <username> <with admin option>;
grant/revoke drop any directory to/from <username> <with admin option>;
grant/revoke read on directory <dir_name> to/from <username> <with grant option>;
grant/revoke write on directory <dir_name> to/from <username> <with grant option>;
--


</div></b>

**Comment by ajit-at-edb** [comment]
> > Please find below sample commands commands
> 
> > > > Should we mention about how to grant/revoke create/drop any directory to/from user/role commands
> > > 
> > > 
> > > @ajit-at-edb I think that's a good idea but I would need some guidance on what instructions to provide.
> > 
> > 
> > Please find below sample commands commands
> > ```
> > grant/revoke create any directory to/from  username <with admin option>;
> > ```
> 
> @ajit-at-edb Do the examples I added to the description of CREATE DIRECTORY and DROP DIRECTORY make sense?

Thanks a lot for adding exmaples. I agree

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**